src/SprykerEco/Zed/ComputopApi/Business/Mapper/ExpressCheckout/AbstractPayPalExpressMapper.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\ComputopApi\Business\Mapper\ExpressCheckout;
+
+use Generated\Shared\Transfer\ComputopApiRequestTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerEco\Service\ComputopApi\ComputopApiServiceInterface;
+use SprykerEco\Shared\ComputopApi\Config\ComputopApiConfig as ComputopApiConstants;
+use SprykerEco\Zed\ComputopApi\ComputopApiConfig;
+
+abstract class AbstractPayPalExpressMapper implements PayPalExpressMapperInterface
+{
+    /**
+     * @var \SprykerEco\Service\ComputopApi\ComputopApiServiceInterface
+     */
+    protected $computopApiService;
+
+    /**
+     * @var \SprykerEco\Zed\ComputopApi\ComputopApiConfig
+     */
+    protected $config;
```

Comment:
```suggestion
    protected $computopApiConfig;
```

to keep naming pattern with service above